kythe/java/com/google/devtools/kythe/platform/shared/ProtobufMetadataLoader.java

```diff
@@ -81,10 +83,11 @@ public ProtobufMetadataLoader(
   /** @return a function that looks up the VName for some filename in the given CompilationUnit. */
   private static Function<String, VName> lookupVNameFromCompilationUnit(CompilationUnit unit) {
     HashMap<String, VName> map = new HashMap<>();
+    Path root = Paths.get(unit.getWorkingDirectory().equals("") ? "/" : unit.getWorkingDirectory());
```

Comment:
`.isEmpty()`

Reply:
Or would `Paths.get("/", unit.getWorkingDirectory())` work?

Reply:
That should work for any valid (non-relative) working directory, done.